src/cmd/cli/command/cd.go

```diff
@@ -18,8 +18,8 @@ var cdCmd = &cobra.Command{
 
 var cdDestroyCmd = &cobra.Command{
 	Use:         "destroy",
-	Annotations: authNeededAnnotation,
-	Args:        cobra.NoArgs, // TODO: set MaximumNArgs(1),
+	Annotations: authNeededAnnotation, // need subscription
```

Comment:
I think this is a TODO comment, and I think this is what it means, but let me know if I'm wrong.
```suggestion
	Annotations: authNeededAnnotation, // TODO: add subscriptionNeededAnnotation
```

Reply:
No, I meant auth is needed because the user needs subscription